lib/__tests__/github.js

```diff
@@ -13,18 +13,16 @@ jest.mock('@octokit/rest', () => {
 
 describe('lib/github', () => {
   describe('isGitHubContributor()', () => {
-    afterEach(() => {
-      mockIssues = {
-        data: { total_count: 0 }
-      };
-    });
-
     test('returns true when the user has merged PRs in the org', async () => {
       const val = await isGitHubContributor('jlengstorf');
       expect(val).toBe(true);
     });
 
     test('returns false when the user has NOT merged any PRs in the org', async () => {
+      mockIssues = {
+        data: { total_count: 0 }
+      };
```

Comment:
This wasn't actually part of this PR — I just couldn't figure out how the hell this test passed when I looked in here. Refactored so it's easier to understand.

Reply:
Ah, so this _was_ (maybe?) passing because it was run before the second test most of the time, but after the first test, hah. Yeah, this is a lot clearer.

Might make sense to re-factor to totally remove the `mockIssues` variable, doesn't seem to be needed for each test!

Reply:
The `afterEach` was doing the same thing as this refactor, but it was overly clever about it.

`mockIssues` is what the `@octokit/rest` `search` call returns for each call, so it needs to be defined at all times. I'm not sure if there's an easier way to mock third-party API calls, but this is what I was able to get working. 😅

Reply:
Few ways!

Here's probably how I'd do it:

```js
jest.mock('@octokit/rest', () => jest.fn());
import octokit from '@octokit/rest';
import { isGitHubContributor } from '../github.mjs';
import mockIssues from './fixtures/github-issues.json';

beforeEach(() => {
  octokit.mockReset();
});

const mockOctokitRestCall = (issues = mockIssues) => octokit.mockReturnValueOnce({
  search: {
    issues: () => issues
  }
});

test(`it mocks something`, () => {
  mockOctokitRestCall(); // then do something after!
});

```

something like that should be a little clearer!